service-front/app/src/Actor/src/Form/RequestActivationKey/RequestReferenceNumber.php

```diff
@@ -20,14 +19,19 @@
  */
 class RequestReferenceNumber extends AbstractForm implements InputFilterProviderInterface
 {
+    private bool $merisEntryEnabled;
+
     public const FORM_NAME = 'request_activation_key_reference_number';
 
     /**
      * RequestActivationKey constructor.
      * @param CsrfGuardInterface $csrfGuard
+     * @param bool $featureFlagEnabled
      */
-    public function __construct(CsrfGuardInterface $csrfGuard)
+    public function __construct(CsrfGuardInterface $csrfGuard, bool $featureFlagEnabled)
     {
+        $this->merisEntryEnabled = $featureFlagEnabled;
```

Comment:
```suggestion
        $this->merisEntryEnabled = $merisEntryEnabled;
```
Just keeping it all consistent with the class variable. Now it's also more self documenting.